src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -131,8 +132,12 @@ def ensure_bicep_installation(cli_ctx, release_tag=None, target_platform=None, s
                 print(f"Installing Bicep CLI {release_tag}...")
             else:
                 print("Installing Bicep CLI...")
-        os.environ.setdefault("CURL_CA_BUNDLE", certifi.where())
-        request = urlopen(_get_bicep_download_url(system, release_tag, target_platform=target_platform))
+        context = ssl.create_default_context(cafile=(
+          os.environ.get("REQUESTS_CA_BUNDLE")
+          or os.environ.get("CURL_CA_BUNDLE")
+          or certifi.where()
+        ))
+        request = urlopen(_get_bicep_download_url(system, release_tag, target_platform=target_platform), context=context)
```

Comment:
Could we consider using `requests.get` instead of `urlopen`, which may make the code more concise and consistent with the approach of `get_bicep_available_release_tags()` and `get_bicep_latest_release_tag()`